docs/src/concepts/using_manifolds.md

```diff
@@ -130,8 +130,14 @@ Retractions are numerically efficient approximations to convert a tangent vector
 
 ### Q7) Projection vs. Log map
 
-Projections are somewhat ambiguous, since on the one hand it might mean that a point from the manifold is projected up onto a nearby tangent space.  
+The term projection can be somewhat ambiguous between references.  In Manifolds.jl, projections either project a point in the [embedding](https://en.wikipedia.org/wiki/Embedding) to a point on the manifold, or a vector from the embedding onto a tangent space at a certain point.  
+
+Confusion, can easily happen since a second type of projection that people sometimes use:  that is when a vector in the ambient space around a manifold (if one exists) is projected down onto a nearby tangent space.
```

Comment:
This is the same thing as the second usage of the word "projection" described in the first paragraph of this answer. Maybe these paragraphs should be joined to avoid an impression that this is yet another meaning of projection?

Reply:
Ah, maybe that is part of it.  By embedding here, do you mean the manifold exists as an embedding inside a larger ambient space?

What I'm trying to keep apart is that the ambient space aspect is not required (I'm following a book by Fortney).  So "that kind of projection" is not possible if there is no ambient space around in which the tangent space happens to occur in.  Does that make sense?

Reply:
```suggestion
Confusion, can easily happen between cases where there is no ambient space around a particular manifold.  Then the term projection may be moot.
```

Reply:
> By embedding here, do you mean the manifold exists as an embedding inside a larger ambient space?

Yes, in a way all manifolds in Manifolds.jl have an embedding in some ambient flat Euclidean space. Note that sometimes all a manifold uses its the ambient space for is representation of points and tangent vectors.

> What I'm trying to keep apart is that the ambient space aspect is not required (I'm following a book by Fortney). So "that kind of projection" is not possible if there is no ambient space around in which the tangent space happens to occur in. Does that make sense?

Mathematicians tend to define manifolds through maps and atlases and in that definition ambient space is not needed. It can be added when necessary. But a computer can only store numbers, and representing a point by its parameters in some map is not always the best idea. Storing parameters vs storing ambient space representation is a deep discussion, though. Even when you store parameters, you can basically (for useful connected manifolds) represent almost every point using a single map. So you can call it (a subset of) a Euclidean space with a custom metric.

Maybe @kellertuer can add some less confusing notes :).

Reply:
Okay, i went looking for the paragraph...  Fortney 2018, p315-p316.  The whole piece is bit long, but the crux for us here is probably this piece:

"""
10.2 Tangent Space of a Manifold

...

We would like to be able to define and thing about tangent spaces without "stepping outside" the manifold.  There are many cases where being able to thing about the manifold properties while still "being inside" the manifold is useful.  A property of a manifold that we can define of think about while still "being inside" the manifold is call[ed] an **intrinsic** property.  An intrinsic property does not, in any way, require or rely on the manifold being embedded into some R^n.  Even though we have been thinking of tangent vectors and tangent spaces extrinsically up to now [but] they are, in fact, intrinsic properties.  We can define them and think about them while still "being inside" the manifold.  When we do this the idea of arrows emanating from a point is no longer appropriate.
"""

...

Reply:
Think it's okay to work with the latest for the Caesar.jl docs page. And this paragraph above is likely out of scope for the purpose of the projections definition.  My sense is that there will be a formal definition somewhere relating to the action of projection with regard to the intrinsic property.  That's the part that tripped me up before.

Reply:
So, hm, concerning tangent spaces and embeddings, one can write half a book ;)

Usually mathematically you would define the tangent space as the collection of all possible derivatives of curves at a point p (or think of all velocities - direction and speed - curves can have). This is for sure something intrinsic to the charts and curves therein. Tangent spaces are intrinsic :)
Mateusz is right in the sense that on a computer we would store thats in numbers, but we could think in charts and partial derivatives of charts for sure - it would just be very complicated to keep track of the chart we are in.

So one usually uses some kind of matrix representation – for example instead of thinking in Euler angles (which would be a parametrisation for the sphere) lets better think of unit vectors (1 x (n+1) matrices), then tangent vectors are usually also some form of a matrix (or vector), again for the sphere vectors that are orthogonal to the point.

In all this we would never think of points being projected onto tangent spaces - the first because we are intrinsic, in the second because we still have in mind that we are to some extend intrinsic – we treat points as if we were intrinsic (see exp/log which really just stay on the manifold.

So projectors are always related to the embedding. For example any point (just not the zero point) in R^(n+1) can be uniquely projected onto the sphere; similarly I can project any point in R^{n+1} onto some tangent space (since that is orthogonal to its base point p, we mainly have to remove the “p-part” of a vector in the embedding.

So _if_ we would like to think of projecting a point from M onto some tangent space, the actual path is
1) take the point p (per se intrinsic, but maybe represented by a matrix in some embedding) and bring it (or think of it as) a point in the embedding
2) project this embedded representation onto the tangent space.

And sure, projection is ambiguous here, since we a) can project from the embedding onto M, and we can project from the embedding onto a tangent space.

Reply:
That's great thanks!  So, I linked from the docs (at the very end of the page where these paragraphs are at) a `note` which links to this conversation.  If anyone is interested and makes it this far, then I'm sure they'll find @mateuszbaran and @kellertuer 's explanations equally insightful!